cmd/cmd/endpointInfo.go

```diff
@@ -45,134 +42,56 @@ Example:
-    //  // token provided with --token (-t) flag
-    //  if exportAPICmdUsername != "" || exportAPICmdPassword != "" {
-    //      // username and/or password provided with -u and/or -p flags
-    //      // Error
-    //      utils.HandleErrorAndExit("username/password provided with OAuth token.", nil)
-    //  } else {
-    //      // token only, proceed with token
-    //  }
-    // } else {
-    //  // no token provided with --token (-t) flag
-    //  // proceed with username and password
-    //  accessToken, apiManagerEndpoint, preCommandErr := utils.ExecutePreCommand(listApisCmdEnvironment, listApisCmdUsername,
-    //      listApisCmdPassword, utils.MainConfigFilePath, utils.EnvKeysAllFilePath)
-
-    //  if preCommandErr == nil {
-    //      if listApisCmdQuery != "" {
-    //          fmt.Println("Search query:", listApisCmdQuery)
-    //      }
-    //      count, apis, err := GetCarbonAppInfo(listApisCmdQuery, accessToken, apiManagerEndpoint)
-
-    //      if err == nil {
-    //          // Printing the list of available APIs
-    //          fmt.Println("Environment:", listApisCmdEnvironment)
-    //          fmt.Println("No. of APIs:", count)
-    //          if count > 0 {
-    //              printAPIs(apis)
-    //          }
-    //      } else {
-    //          utils.Logln(utils.LogPrefixError+"Getting List of APIs", err)
-    //      }
-    //  } else {
-    //      utils.HandleErrorAndExit("Error calling '"+listCmdLiteral+" "+apisCmdLiteral+"'", preCommandErr)
-    //  }
-    // }
-}
-
-// GetEndpointInfo
-// @param name of the endpoint
-// @return Endpoint object
-// @return error
-func GetEndpointInfo(name string) (*utils.Endpoint, error) {
-
-    finalUrl := utils.RESTAPIBase + utils.PrefixEndpoints + "?endpointName=" + name
-
-    utils.Logln(utils.LogPrefixInfo+"URL:", finalUrl)
-
-    headers := make(map[string]string)
-    // headers[utils.HeaderAuthorization] = utils.HeaderValueAuthPrefixBearer + " " + accessToken
-
-    resp, err := utils.InvokeGETRequest(finalUrl, headers)
-
-    if err != nil {
-        utils.HandleErrorAndExit("Unable to connect to "+finalUrl, err)
-    }
-
-    utils.Logln(utils.LogPrefixInfo+"Response:", resp.Status())
-
-    if resp.StatusCode() == http.StatusOK {
-        endpointResponse := &utils.Endpoint{}
-        unmarshalError := xml.Unmarshal([]byte(resp.Body()), &endpointResponse)
-
-        if unmarshalError != nil {
-            utils.HandleErrorAndExit(utils.LogPrefixError+"invalid XML response", unmarshalError)
-        }
-
-        return endpointResponse, nil
-    } else {
-        return nil, errors.New(resp.Status())
-    }
-
 }
 
 // printEndpointInfo
 // @param Endpoint : Endpoint object
 func printEndpoint(endpoint utils.Endpoint) {
 
-	table := tablewriter.NewWriter(os.Stdout)
+    table := tablewriter.NewWriter(os.Stdout)
 
-	d_name := []string{"NAME", endpoint.Name}
-	table.Append(d_name)
+    d_name := []string{"NAME", endpoint.Name}
```

Comment:
Are we using the correct naming convention for variables? AFAIK normally we use camelcase.

Reply:
In `endpointList.go` you have used camelcase. Shall we use one convention?